sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_default_storage_helper.py

```diff
@@ -1,343 +0,0 @@
-# ---------------------------------------------------------
```

Comment:
I couldn't see where/how this file was being used - so I removed it. Happy to add it back if my housekeeping was a bit over-enthusiastic :)